scripts/modulesort.js

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 // This script returns a list of the modules
-// ordered by checking their dependencies
+// ordered by checking their dependencies 
```

Comment:
you are introducing here a space with no reason ^^

Reply:
Yep, I needed to update just the submodule core and did this -.-. Updating in the next commit